tests/net/icmpv4/prj.conf

```diff
@@ -13,5 +13,5 @@ CONFIG_NET_LOG=y
 CONFIG_ENTROPY_GENERATOR=y
 CONFIG_TEST_RANDOM_GENERATOR=y
 CONFIG_ZTEST=y
-CONFIG_MAIN_STACK_SIZE=1280
+CONFIG_MAIN_STACK_SIZE=2048
```

Comment:
This will affect all the boards even if they do not need a stack this large. I just wonder could we increase the stack only for SPARC.

Reply:
Yes I agree that it would be the preferred solution, but I am unfortunately not aware a good way to achieved that. In `arch/sparc/Kconfig`, the `MAIN_STACK_SIZE` is already set to `2048`, but is overridden by the test `prj.conf` here. And if we just remove the `CONFIG_MAIN_STACK_SIZE` in `prj.conf`, then lots of other boards may start to fail. Do you have any recommendation?

Reply:
One option is to create board specific prj.conf to `tests/../../boards/<board-name>.conf` and override the option there, but as there are multiple boards with this arch, it is not very scalable solution.
For ztest stack size, there is the `CONFIG_TEST_EXTRA_STACKSIZE` where the stack is increased for some platforms. Perhaps this solution could be extented to main stack too, so a new option like `CONFIG_MAIN_EXTRA_STACK_SIZE` could be introduced. This option could then be set in the arch defaults.

Reply:
> In arch/sparc/Kconfig, the MAIN_STACK_SIZE is already set to 2048

This seems like pretty rough way. What if some application needs more than 2K of stack?

The overall question - is this "SPARC" in the PR title a 64-bit SPARC? Then it's not surprise that it requires more stack, any 64-bit platform would be like that, and we need to find out generic solution for all 64-bit platforms.

And if not, then why such a big jump from 1280 to 2048? Are you sure it won't work with 1380? And if it is, we can just got for that (we regularly bump that anyway, as Zephyr stack usage constantly grows).

Reply:
Thank you for the input @jukkar and @pfalcon

I have revisited the `tests/net` stack changes. It turned out that 64 byte increase was actually enough, except for one test which needed 96 byte and one at 192 byte. The PR has been updated to use these numbers.

Is this an acceptable solution for you for now?

Reply:
Yes, from my PoV, it's ok, thanks.